action.yml

```diff
@@ -1,5 +1,5 @@
-name: upload-openapi-spec
-description: Upload your OpenAPI spec to update your SDKs (and, if configured, add example snippets to your API docs). stainlessapi.com
+name: stainless-upload-openapi-spec
```

Comment:
Hmm, looking at other actions we do not need to use an hypenasized name. I will change to something nicer.